build/downgrade-rector.sh

```diff
@@ -40,4 +40,4 @@ done
 
 # CONFIRMED: give time to print all the files, before the next process takes over newly printed content
 # avoids bugs like these half of files done, next half waiting https://github.com/rectorphp/rector-src/runs/2565478682
-sleep 20
+sleep 25
```

Comment:
Ref https://github.com/rectorphp/rector-src/pull/277#issuecomment-866952510

Reply:
@TomasVotruba it stil error:

```bash
bin/rector -c build/config/config-downgrade.php --ansi process src/Configuration/ConfigurationFactory.php --dry-run --debug
[parsing] src/Configuration/ConfigurationFactory.php
[refactoring] src/Configuration/ConfigurationFactory.php
    [applying] Rector\DowngradePhp74\Rector\LNumber\DowngradeNumericLiteralSeparatorRector
    [applying] Rector\Removing\Rector\Class_\RemoveInterfacesRector
    [applying] Rector\DowngradePhp80\Rector\Class_\DowngradeAttributeToAnnotationRector
    [applying] Rector\DowngradePhp80\Rector\Class_\DowngradePropertyPromotionRector
    [applying] Rector\DowngradePhp72\Rector\Class_\DowngradeParameterTypeWideningRector
    [applying] Rector\DowngradePhp80\Rector\Property\DowngradeUnionTypeTypedPropertyRector
    [applying] Rector\DowngradePhp74\Rector\Property\DowngradeTypedPropertyRector
    [applying] Rector\DowngradePhp80\Rector\FunctionLike\DowngradeUnionTypeDeclarationRector
    [applying] Rector\DowngradePhp80\Rector\FunctionLike\DowngradeMixedTypeDeclarationRector
    [applying] Rector\DowngradePhp80\Rector\ClassMethod\DowngradeStaticTypeDeclarationRector
    [applying] Rector\DowngradePhp80\Rector\ClassMethod\DowngradeTrailingCommasInParamUseRector
    [applying] Rector\DowngradePhp74\Rector\ClassMethod\DowngradeCovariantReturnTypeRector
    [applying] Rector\DowngradePhp74\Rector\ClassMethod\DowngradeContravariantArgumentTypeRector
    [applying] Rector\DowngradePhp74\Rector\ClassMethod\DowngradeSelfTypeDeclarationRector
    [applying] Rector\DowngradePhp72\Rector\FunctionLike\DowngradeObjectTypeDeclarationRector
    [applying] Rector\DowngradePhp80\Rector\Expression\DowngradeMatchToSwitchRector
    [applying] Rector\DowngradePhp80\Rector\MethodCall\DowngradeNamedArgumentRector
    [applying] Rector\DowngradePhp73\Rector\FuncCall\DowngradeTrailingCommasInFunctionCallsRector
    [applying] Rector\DowngradePhp80\Rector\ClassConstFetch\DowngradeClassOnObjectToGetClassRector
PHP Fatal error:  Uncaught Rector\Core\Exception\NotImplementedYetException in /Users/samsonasik/www/rector-src/rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php:224
Stack trace:
#0 /Users/samsonasik/www/rector-src/rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php(178): Rector\DowngradePhp80\Rector\MethodCall\DowngradeNamedArgumentRector->mapPHPStanTypeToExpr(Object(PHPStan\Type\Constant\ConstantBooleanType))
#1 /Users/samsonasik/www/rector-src/rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php(157): Rector\DowngradePhp80\Rector\MethodCall\DowngradeNamedArgumentRector->replacePreviousArgs(Object(PhpParser\Node\Expr\New_), Array, 7, Array)
#2 /Users/samsonasik/www/rector-src/rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php(111): Rector\DowngradePhp80\Rector\MethodCall\DowngradeNamedArgumentRector->processRemoveNamedArgument(Object(PHPStan\Reflection\ResolvedMethodReflection), Object(PhpParser\Node\Expr\New_), Array)
#3 /Users/samsonasik/www/rector-src/rules/DowngradePhp80/Rector/MethodCall/DowngradeNamedArgumentRector.php(96): Rector\DowngradePhp80\Rector\MethodCall\DowngradeNamedArgumentRector->removeNamedArguments(Object(PhpParser\Node\Expr\New_), Array)
#4 /Users/samsonasik/www/rector-src/src/Rector/AbstractRector.php(247): Rector\DowngradePhp80\Rector\MethodCall\DowngradeNamedArgumentRector->refactor(Object(PhpParser\Node\Expr\New_))
#5 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(123): Rector\Core\Rector\AbstractRector->enterNode(Object(PhpParser\Node\Expr\New_))
#6 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Stmt\Return_))
#7 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#8 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Stmt\ClassMethod))
#9 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#10 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Stmt\Class_))
#11 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(114): PhpParser\NodeTraverser->traverseArray(Array)
#12 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(223): PhpParser\NodeTraverser->traverseNode(Object(PhpParser\Node\Stmt\Namespace_))
#13 /Users/samsonasik/www/rector-src/vendor/nikic/php-parser/lib/PhpParser/NodeTraverser.php(91): PhpParser\NodeTraverser->traverseArray(Array)
#14 /Users/samsonasik/www/rector-src/src/PhpParser/NodeTraverser/RectorNodeTraverser.php(41): PhpParser\NodeTraverser->traverse(Array)
#15 /Users/samsonasik/www/rector-src/src/Application/FileProcessor.php(38): Rector\Core\PhpParser\NodeTraverser\RectorNodeTraverser->traverse(Array)
#16 /Users/samsonasik/www/rector-src/src/Application/FileProcessor/PhpFileProcessor.php(147): Rector\Core\Application\FileProcessor->refactor(Object(Rector\Core\ValueObject\Application\File))
#17 /Users/samsonasik/www/rector-src/src/Application/FileProcessor/PhpFileProcessor.php(164): Rector\Core\Application\FileProcessor\PhpFileProcessor->Rector\Core\Application\FileProcessor\{closure}(Object(Rector\Core\ValueObject\Application\File))
#18 /Users/samsonasik/www/rector-src/src/Application/FileProcessor/PhpFileProcessor.php(146): Rector\Core\Application\FileProcessor\PhpFileProcessor->tryCatchWrapper(Object(Rector\Core\ValueObject\Application\File), Object(Closure), 'refactoring', Object(Rector\Core\ValueObject\Configuration))
#19 /Users/samsonasik/www/rector-src/src/Application/FileProcessor/PhpFileProcessor.php(78): Rector\Core\Application\FileProcessor\PhpFileProcessor->refactorNodesWithRectors(Array, Object(Rector\Core\ValueObject\Configuration))
#20 /Users/samsonasik/www/rector-src/src/Application/ApplicationFileProcessor.php(52): Rector\Core\Application\FileProcessor\PhpFileProcessor->process(Array, Object(Rector\Core\ValueObject\Configuration))
#21 /Users/samsonasik/www/rector-src/src/Application/ApplicationFileProcessor.php(34): Rector\Core\Application\ApplicationFileProcessor->processFiles(Array, Object(Rector\Core\ValueObject\Configuration))
#22 /Users/samsonasik/www/rector-src/src/Console/Command/ProcessCommand.php(127): Rector\Core\Application\ApplicationFileProcessor->run(Array, Object(Rector\Core\ValueObject\Configuration))
#23 /Users/samsonasik/www/rector-src/vendor/symfony/console/Command/Command.php(299): Rector\Core\Console\Command\ProcessCommand->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 /Users/samsonasik/www/rector-src/vendor/symfony/console/Application.php(978): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#25 /Users/samsonasik/www/rector-src/vendor/symfony/console/Application.php(295): Symfony\Component\Console\Application->doRunCommand(Object(Rector\Core\Console\Command\ProcessCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#26 /Users/samsonasik/www/rector-src/src/Console/ConsoleApplication.php(79): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#27 /Users/samsonasik/www/rector-src/vendor/symfony/console/Application.php(167): Rector\Core\Console\ConsoleApplication->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#28 /Users/samsonasik/www/rector-src/bin/rector.php(60): Symfony\Component\Console\Application->run()
#29 /Users/samsonasik/www/rector-src/bin/rector(4): require_once('/Users/samsonas...')
```

Reply:
@TomasVotruba should be fixed at https://github.com/rectorphp/rector-src/pull/279 : 

![Screen Shot 2021-06-23 at 23 42 00](https://user-images.githubusercontent.com/459648/123135967-9ec9f180-d47c-11eb-8d47-228e3286f782.png)